cache/manager.go

```diff
@@ -1066,14 +1066,32 @@ func (cm *cacheManager) pruneOnce(ctx context.Context, ch chan client.UsageInfo,
 		}
 	}
 
-	return cm.prune(ctx, ch, pruneOpt{
+	var (
+		totalReleasedSize  int64
+		totalReleasedCount int64
+	)
+
+	popt := pruneOpt{
 		filter:       filter,
 		all:          opt.All,
 		checkShared:  check,
 		keepDuration: opt.KeepDuration,
 		keepBytes:    calculateKeepBytes(totalSize, dstat, opt),
 		totalSize:    totalSize,
-	})
+	}
+	for {
+		releasedSize, releasedCount, err := cm.pruneOnce(ctx, ch, popt)
```

Comment:
`releasedCount` in here seems more like a bool indicating if anything was pruned (unless I'm missing some usage of `totalReleasedCount`).